UserTools/DigitBuilder/DigitBuilder.cpp

```diff
@@ -57,6 +59,7 @@ bool DigitBuilder::Initialise(std::string configfile, DataModel &data){
     Log("DigitBuilder Tool: Error retrieving Geometry from ANNIEEvent!",v_error,verbosity); 
     return false; 
   }
+  std::cout << "Strip Hit Mode:" << striphit << endl;
```

Comment:
please remove cout statements. Use Log with suitable logging level for debug.

Reply:
i won't comment on them all but cout is used multiple times here.

Reply:
Noted.  I'll go through and replace them.